### DISCUSSION TITLE
Turn off light from PS4 controller

### DISCUSSION THREAD

**Discussion by Hendrix900** [darthcloud/BlueRetro#838] [General]
Hello,
I would like to know if it is possible to turn off the PS4 controller light from the BlueRetro settings.
Light reflects off the CRT and is not seen clearly.

Thank you!

**Comment by darthcloud** [comment]
Currently not. I might make that customizable in the future.

For now the only thing you can do is put tape on it ;)

**Comment by konwektor** [comment]
Or comment out led behavior in source  - bt hid ps.c file i think - somewhere in case : blabla, swittch blabla.
Or use 
[https://ds4-windows.com/](url) for customizing setting. There is an option for switching.changing ligths. maybe will not activate lajter in use with BR
